_posts/2023-06-08-opensearch-project-newsletter-volume1-issue4.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "OpenSearch Project Newsletter - Volume 1, Issue 4"
+authors: 
+  - pattijuric
+date: 2023-06-08
+meta_keywords: OpenSearch Project news, open source newsletter, OpenSearch partner
+meta_description: Hear what’s new with our partners and learn about the latest news, community happenings, product launches, and events when your read the OpenSearch Project Newsletter.
+twittercard:
+  description: Hear what’s new with our partners and learn about the latest news, community happenings, product launches, and events when your read the OpenSearch Project Newsletter.
+category:
+  - community
+excerpt: 
+---
+
+The OpenSearch Project is pleased to present our newsletter. Distributed monthly, the newsletter is a great way to learn more about the latest OpenSearch news, relevant content from the community, and upcoming events that you may be interested in. If you have topic or something you’d like us to share in the newsletter, please reach out to [Patti Juric](mailto:pattijur@amazon.com)
+
+![opensearch newsletter volume 1 issue 4](/assets/media/blog-images/2023-06-08-opensearch-newsletter-volume1-issue4/newslettervol4.png){:class="img-fluid"}
+## What’s New?
+
+**[OpenSearch 2.8.0 is here](https://opensearch.org/blog/opensearch-2.8.0-released/),** and it has a host of new features and enhancements and experimental functionality that are sure to generate excitement within the community! This release includes new tools for machine learning, search, and observability workloads and several enhancements to better manage your indexes and data. [Check out what’s new in 2.8.0](https://opensearch.org/blog/opensearch-2.8.0-released/). 
+
+[**Data Prepper 2.3.0 is now available for download!**](https://opensearch.org/downloads.html#data-prepper) This release introduces a number of changes that improve Data Prepper’s ability to process complex expressions with functions, arithmetic operations, and string operations. [Get started with it now.](https://opensearch.org/downloads.html#data-prepper)  
```

Comment:
Data Prepper 2.3.0 headline should link here: https://opensearch.org/blog/Announcing-Data-Prepper-2.3.0/